README.md

```diff
@@ -62,9 +62,11 @@ The SDK bundle contains the following folders:
 The SDK currently supports and is tested on the following OS/Compiler variations:
 
 - 64bit Linux with GCC 14
+- Arm64 Linux with GCC 14
```

Comment:
Okay, sorry, but if we do that, let's write `amd64` or `x64` where we currently say 64bit and place `x64`/`arm64` on the same line.

Reply:
Is this better?
```
- x64/arm64 Linux with GCC 14
- x64 Linux with GCC 12
- x64 Linux with GCC 9
- x64/arm64 Linux with clang 19
- x86 Linux with GCC 9 (cross compiled from x64 host)
- x86 Windows with MSVC 2019
- x64 Windows with MSVC 2022
```